### PR TITLE
Ex201 - Exercise 01 chapter Array

### PR DESCRIPTION
**Pull request by hobathanh**
create and add Exercise201

### CODE REVIEW

File: src/main/java/com/bathanh/exercise/basic/array/Exercise201.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.bathanh.exercise.basic.array;
+
+public class Exercise201 {
+
+    public int searchArrayLargest(int[] arrN) {
```
Comment (voquanghoa) [inline src/main/java/com/bathanh/exercise/basic/array/Exercise201.java:5]:
```suggestion
    public int searchArrayLargest(final int[] array) {
```

File: src/main/java/com/bathanh/exercise/basic/array/Exercise201.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        int maxArrN = arrN[0];
```
Comment (voquanghoa) [inline src/main/java/com/bathanh/exercise/basic/array/Exercise201.java:6]:
```suggestion
        int max = arrN[0];
```

File: src/main/java/com/bathanh/exercise/basic/array/Exercise201.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        for (int i = 0; i <= arrN.length - 1; i++) {
```
Comment (voquanghoa) [inline src/main/java/com/bathanh/exercise/basic/array/Exercise201.java:7]:
```suggestion
        for (int i = 0; i < arrN.length; i++) {
```

File: src/main/java/com/bathanh/exercise/basic/array/Exercise201.java (hunk 1 above)
Comment (hobathanh) [reply src/main/java/com/bathanh/exercise/basic/array/Exercise201.java]: resolved

File: src/main/java/com/bathanh/exercise/basic/array/Exercise201.java (hunk 3 above)
Comment (hobathanh) [reply src/main/java/com/bathanh/exercise/basic/array/Exercise201.java]: resolved

**Pull request merged**